extensions/identity-hub-client/build.gradle.kts

```diff
@@ -57,15 +29,35 @@ sourceSets {
 }
 
 val jacksonVersion: String by project
+val okHttpVersion: String by project
+val rsApi: String by project
+val edcVersion: String by project
+val edcGroup: String by project
+val jupiterVersion: String by project
+val restAssured: String by project
+val faker: String by project
+val assertj: String by project
 
-// Dependencies copied from build/generate-resources/main/build.gradle
 dependencies {
+    implementation("${edcGroup}:core:${edcVersion}")
```

Comment:
why the dependencies on EDC core, and the dependencies on runtime module `:extensions:identity-hub`?

It seems to me the `IdentityHubClient` does not need to be an EDC extension, could reside outside of the `extensions` structure. This way it can also be used by the Identity Hub CLI, which is not an EDC runtime.

Reply:
I moved it and removed the EdcException usage to remove the edc core dependency, now the client throws IOException and JsonProcessingException